tests/testthat/test-plot-coordmap.R

```diff
@@ -357,3 +368,51 @@ test_that("ggplot coordmap with various scales and coords", {
     sortList(list(left=-1, right=3, bottom=-2, top=4))
   )
 })
+
+
+test_that("ggplot coordmap maintains discrete limits", {
+  tmpfile <- tempfile("test-shiny", fileext = ".png")
+  on.exit(rm(tmpfile))
```

Comment:
`unlink`, not `rm`. And thanks for fixing the other places that have this same error. :)